app/src/main/java/studyBuddy/timemanagement/EndSessionButtonListener.java

```diff
@@ -0,0 +1,67 @@
+package studyBuddy.timemanagement;
+
+import android.animation.AnimatorInflater;
+import android.animation.ObjectAnimator;
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+import android.view.MotionEvent;
+import android.view.View;
+
+import com.example.studdybuddy.R;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import studyBuddy.Session;
+
+// TODO: change onTouchListener to onClickListener
+public class EndSessionButtonListener implements View.OnTouchListener {
```

Comment:
does this need to be done

Reply:
not atm, there's some additional functionality to handle the difference between the two that we can remove by making the change